src/ophys_etl/workflows/db/schemas.py

```diff
@@ -203,6 +203,33 @@ def _generate_binary_mask(
             ] = True
         return mask.tolist()
 
+    def is_valid(self, include_decrosstalk: bool = False) -> bool:
+        """Return True if no processing flags (motion_border, decrosstalk)
+        flags are set.
+
+        Parameters
+        ----------
+        include_decrosstalk : bool Optional
+            Check decrosstalk flags for is_valid. Decrosstalk is required
+            for mesoscope experiments imaging multiple experiments at once.
+            Other experiments using different equipment do not require
+            decrosstalk.
+
+        Returns
+        -------
+        is_valid : bool
+            Composite flag that returns True if no processing flags are set.
+            False if any flag is set.
+        """
+        flag_set = [self.is_in_motion_border]
+        if include_decrosstalk:
+            flag_set += [self.is_decrosstalk_ghost,
+                         self.is_decrosstalk_invalid_raw_active,
+                         self.is_decrosstalk_invalid_raw,
+                         self.is_decrosstalk_invalid_unmixed,
+                         self.is_decrosstalk_invalid_unmixed_active]
+        return sum(flag_set) == 0
```

Comment:
There is a bug here actually. Please parameterize the equipment name in your test and you will see the bug. 

```
TypeError                                 Traceback (most recent call last)
<ipython-input-8-ebd9144afb38> in <module>
----> 1 sum([None, True])

TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
```